authorization/index.md

```diff
@@ -0,0 +1,90 @@
+# gNSI Authorization Protobuf Definition.
+**Contributors**: hines@google.com, morrowc@google.com, tmadejski@google.com
+**Last Updated**: 2022-07-12
+
+## Background
+
+The authorization proto definition provides a clear method to define
+and implement policy authorizing user or group access to the management
+systems and interfaces of a network deployment. The ability to permit
+or deny access to these systems and interfaces is intended to help
+operate a network in a more safe and secure manner.
+
+## Motivation
+
+Authorization services for network systems can, at times, be operated
+as remote services. One example of this is a TACACS+ service, there are
+inherent challenges with operating a network and relying upon a remote
+service for critical functions performed on network systems. A solution
+to provide on-device Authorization with the fidelity of remote services
+should be provided.
+
+## Architecture
+
+There is no requirement for the Authorization policy to be evaluated:
+on the local network system, in a microservice operated on the network
+system, or in every gNMI service individually on the network system.
+It is expected that gNMI services enabled on a network system respect
+the AuthorizationPolicy installed, however.
+
+#### Pathz.Install()
+
+Pathz.Install() will permit installation, and verification of function,
+of an AuthorizationPolicy. The normal use-case would be to:
+
+* send an AuthorizationPolicy{} to a network system as an
+InstallPathzRequest{}
+* verify access/authorization has changed to the desired state
+through existing gNMI methods, or with pathz.Probe() requests.
+* send a FinalizeRequest{} to finish the installation process.
+
+#### Pathz.Rotate()
+
+Pathz.Rotate() will permit rotation, and verification of function,
+of an AuthorizationPolicy. The normal use-case would be to:
+
+* send an AuthorizationPolicy{} to a network system as a
+RotatePathzRequest{}
+* verify access/authorization has changed to the desired state
+through existing gNMI methods, or with pathz.Probe() requests.
+* send a FinalizeRequest{} to finish the installation process.
+
+#### Pathz.Probe()
+
+Pathz.Probe() provides a method to test the AuthorizationPolicy
+with a ProbeRequest{} which includes a user and gNMI path. This
+enables network operations and management systems to verify that
+the access expected is either permitted or denied in accordance
+with the expected to be deployed AuthroizationPolicy.
+
+## User Experiences
+
+#### An AuthorizationPolicy is to be installed.
```

Comment:
one too many indents?

Reply:
i was literally copying your `healthz` :) 
but, also.. I suck very much at markdown :( so.. yea, happy to adjust :)